From 3909877f12e50556e844bc20e72870a4fa905ada Mon Sep 17 00:00:00 2001
From: James Chapman <jachapma@redhat.com>
Date: Tue, 9 Nov 2021 12:55:28 +0000
Subject: [PATCH 10/12] Issue 4997 - Function declaration compiler error on
 1.4.3

Bug description: Building the server on the 1.4.3 branch generates a
compiler error due to a typo in function declaration.

Fixes: https://github.com/389ds/389-ds-base/issues/4997

Reviewed by: @jchapman (one line commit rule)
---
 ldap/servers/slapd/slapi-private.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/slapd/slapi-private.h b/ldap/servers/slapd/slapi-private.h
index 570765e47..d6d74e8a7 100644
--- a/ldap/servers/slapd/slapi-private.h
+++ b/ldap/servers/slapd/slapi-private.h
@@ -273,7 +273,7 @@ void *csngen_register_callbacks(CSNGen *gen, GenCSNFn genFn, void *genArg, Abort
 void csngen_unregister_callbacks(CSNGen *gen, void *cookie);
 
 /* debugging function */
-void csngen_dump_state(const CSNGen *gen);
+void csngen_dump_state(const CSNGen *gen, int severity);
 
 /* this function tests csn generator */
 void csngen_test(void);
-- 
2.31.1

